api/src/lastProcessed.py

```diff
@@ -13,27 +13,30 @@ def lastProcessed():
     # get state parameter
     if 'state' in request.args:
         state = request.args['state']
-        state = state.lower()
+        state = state.upper()
     else:
-        return "ERROR: No state specified"
+        return "404 bad request"
 
     # get election_dt parameter
     if 'election_dt' in request.args:
-        election_dt = datetime.strptime(request.args['election_dt'], '%m/%d/%y')
+        election_dt = datetime.strptime(request.args['election_dt'], '%m-%d-%Y')
     else:
-        return "ERROR: No election date provided"
+        return "404 bad request"
 
     # parse the config file
     config = configparser.ConfigParser()
-    if not config.read('config.ini'):
+    if not config.read('../../config.ini'):
```

Comment:
I think we want to keep this as 'config.ini' - this should've just been a change local to your machine (i.e. you can unstage this specific chunk in the future). Just remember to change it back before you commit

Reply:
Gotcha...just forgot to change it back